src/components/Layout/Layout.test.js

```diff
@@ -1,20 +0,0 @@
-import '@testing-library/jest-dom/extend-expect';
```

Comment:
Why remove the test file?

Reply:
I realised this file was a duplicate, I had two same files.